### ISSUE TITLE
Hyperparameters to replicate EfficientNet-B4 Accuracy

### ISSUE THREAD

**Issue by anirban-ghosh** [tensorflow/tpu#884]
Hi, 
I'm working on reproducing the 82.9% accuracy on EfficientNet-B4 using autoaugmentation.  Neither the default configs nor the TPU-EfficientNet tutorial have a set of hyperparameters that can replicate the stated accuracy for EfficientNet-B4.  Has anyone been able to reproduce this, and if so, then with what hyperparameters? 

Thanks, 
Anirban
